src/api/app/assets/javascripts/webui/user_profile.js

```diff
@@ -5,3 +5,27 @@ function moveInvolvementToContainer() { // jshint ignore:line
 
   container.prepend($('#involvement'));
 }
+
+function updateCharactersCount(e) {
+  var maxLength = $(e.target).attr('maxlength');
+  var currentLength = $(e.target).val().length;
+  var remainingCount = maxLength - currentLength;
+
+  // Change font color to "danger" when low amount of characters left
+  $('#bio-chars-counter').toggleClass('text-danger', remainingCount < 10);
+
+  // Make text singular or plural based on the amount of characters
+  var plural = (remainingCount === 1) ? '' : 's';
+  $('#bio-chars-counter').css("visibility", "visible").text(remainingCount + " character" + plural + " remaining");
+}
+
+$(document).ready(function(){
```

Comment:
@OleksandrOrlov usually we try to call the javascript functions from the corresponding view, to avoid the overhead of calling the code everywhere. Its not wrong, but this way you can also avoid interference, incase you used the same html id in another view. If you grep for example for the `moveInvolvementToContainer()` function, you will see how it called from the haml file

Reply:
Good point! Thank you for the note. I've moved the JS to haml file.

I've moved it to `show` file, as the same javascript is needed for both `_edit_dialog` and `_edit_account_form`.